src/fiatExchanges/FiatExchangeAmount.tsx

```diff
@@ -71,7 +67,7 @@ type Props = RouteProps
 
 function FiatExchangeAmount({ route }: Props) {
   const { t } = useTranslation()
-  const { currency, flow, network, tokenId } = route.params
+  const { flow, tokenId, symbol } = route.params
```

Comment:
Here, `symbol` could be found from `useTokenInfo(tokenId)`, however this param is needed later in the `FiatExchangeAmount.navOptions` function which cannot call that hook, so it can also be used here.